app/models/conference.rb

```diff
@@ -15,4 +17,15 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  private
+
+    def date_interval
+      if [starts_on, ends_on].count(&:nil?) == 1
+        errors.add(:base, 'Set both dates or none')
+      elsif starts_on.to_s > ends_on.to_s
```

Comment:
`starts_on` should be less than `ends_on`, right?

Comparing strings lexically is a hack that sure makes it work. But in that case, it should also work if either is `nil`, no?

What about this:
```ruby
unless [starts_on, ends_on].all?(:present?) && (starts_on < ends_on)
  errors.add(:ends_on, 'must be later than start date')
end
```